webpack.common.js

```diff
@@ -1,3 +1,6 @@
+// Copyright (c) 2022. Consensys Software Inc. All rights reserved.
+// Licensed under the MIT license.
+
```

Comment:
Thanks for taking care of the _Copyright Notice_. But given that it is a significant change, should we open a different PR for this?

Reply:
I think this was done by my IDE. I thought this was how all our JS files were marked? Intellij adds in copyrights based on your template by default on all unmarked files.

I can revert all this if you think best?

Reply:
Got it. If it not a big issue reverting, I believe it's better to create another PR to tackle this copyright thing.